public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/cron.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\{get_email_id, queue_survey_email};
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\{disable_page, publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+const TEMP_ATTENDEE_ID   = 'organizer_debrief_survey_temp_attendee';
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_organizer_debrief_survey_email', __NAMESPACE__ . '\queue_organizer_survey' );
+add_action( 'wc_organizer_disable_debrief_survey', __NAMESPACE__ . '\disable_organizer_survey' );
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
+}
+
+/**
+ * Get Wordcamp attendee ID.
+ *
+ * @return int
+ */
+function get_wordcamp_attendees_id() {
+	return get_option( TEMP_ATTENDEE_ID );
+}
+
+/**
+ * Add cron jobs to the schedule.
+ */
+function schedule_jobs() {
+	if ( ! wp_next_scheduled( 'wc_organizer_debrief_survey_email' ) ) {
+		wp_schedule_event( time(), 'daily', 'wc_organizer_debrief_survey_email' );
+	}
+}
+
+/**
+ * Add temporary attendee since email can only be sent to items in attendee tracker.
+ */
+function add_temp_attendee() {
+	if ( ! get_wordcamp_attendees_id() ) {
+		$attendees_id = wp_insert_post( array(
+			'post_title' => 'Organizer debrief survey',
```

Comment:
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/a9a5171e-5d5b-4ea6-9381-c9d1aeb405b0)
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/cfe52659-c476-4278-8e7d-0e509e40d53d)


The title is `(no title)` even though the `post_title` is specified and updated to DB. And after moving it to the Trash, the Title appears. I'm not entirely sure why this happens, but it doesn't affect the overall process.